ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Anyway, these are new scripts which does not have to be perfect right away. People can play with it and open issues/PR for improvements. Just wanted to prepare a general template which can be filled in remaining roles.